package.json

```diff
@@ -10,7 +10,7 @@
     "smoketest": "yarn type-check && yarn lint && yarn format:check && yarn build",
     "type-check": "tsc --noEmit",
     "analyze": "ANALYZE=true yarn build",
-    "lint": "eslint . --ext .js,.jsx,.ts,.tsx --max-warnings 0",
+    "lint": "eslint . --ext .js,.jsx,.ts,.tsx --max-warnings 0 --report-unused-disable-directives",
```

Comment:
This flag will complain when an `eslint-disable` directive doing nothing is found.

When we upgrade to ESLint 8, it will even remove the unused directives when running the command  `lint:fix`

Reply:
Really convenient, right? Looking forward to auto fix it in ESLint v8.

Reply:
I've opened a PR on the Renovate bot repo with this change as well, seems too nice to not use it! 😄